.github/workflows/publish-npm-rc.yaml

```diff
@@ -0,0 +1,62 @@
+name: Publish release candidate
+
+on:
+  push:
+    branches: ['releases/[0-9]+.[0-9]+.[0-9]+-rc[0-9]+']
+
+jobs:
+  build:
+    if: github.repository == 'redwoodjs/redwood'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # Required because lerna uses tags to determine the version.
+
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile --check-files
+
+      - name: Extract version from branch
+        shell: bash
+        run: echo "##[set-output name=version;]$(echo ${GITHUB_REF##*/})"
+        id: extract_version
+
+      - name: Create package version
+        run: ./tasks/update-package-versions ${{ steps.extract_version.outputs.version }}
+
+      - name: Configure git credentials
+        run: |
+          git config user.name redwood-releases
+          git config user.email releases@redwoodjs.com
+      - name: Commit and tag
+        run: |
+          git commit -am "${{ steps.extract_version.outputs.version }}"
+          git tag ${{ steps.extract_version.outputs.version }}
+          git push && git push --tags
+      - name: Build
+        run: yarn build
+
+      - name: Run ESLint
+        run: yarn lint
+        env:
+          CI: true
+
+      - name: Run tests
+        run: yarn test
+        env:
+          CI: true
+
+      - name: Configure NPM credentials
+        run: |
+          echo "//registry.npmjs.org/:_authToken=${NPM_AUTH_TOKEN}" > .npmrc
+        env:
+          NPM_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+
+      - name: Publish package
+        run:
+          yarn lerna publish from-package --yes
```

Comment:
Not an expert on publishing from lerna but how about
```
yarn lerna publish --canary --preid rc --dist-tag rc --yes
```
And we don't have to worry about bumping versions based on the branch name. What do you think?